src/app/(proper_react)/(redesign)/(public)/how-it-works/HowItWorksView.tsx

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
```

Comment:
In this case, the page might benefit from not marking the entire content as client component. We can make the header and footer sections server components. In order to be able to do that we would only need to mark `SectionOne` and `SectionTwo` with `"use client"` and use the `useL10n` hook. For the view, we get `l10n` in `page` with `getL10n` and pass it as a prop.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/4509cb9a709d427714952b724655ff7dbdceb8b2